addons/account/tests/test_account_move_tax_block_date.py

```diff
@@ -0,0 +1,94 @@
+from odoo.addons.account.tests.account_test_classes import AccountingTestCase
+from odoo.exceptions import ValidationError, UserError
+from datetime import datetime, timedelta
+from dateutil.relativedelta import relativedelta
+from calendar import monthrange
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class TestTaxBlockDate(AccountingTestCase):
+    """
+    Forbid creation, edition and deletion of Journal Items out with a date prior
+    to the Tax Block Date
+    """
+
```

Comment:
I think, each of your tests should have a proper docstring depicting what's going to be tested